packages/core/src/cli/prepare.ts

```diff
@@ -12,6 +12,9 @@ function initNodeEnv() {
 export function prepareCli(): void {
   initNodeEnv();
 
+  // make it easier to identify the process via activity monitor or other tools
+  process.title = 'rsbuild-node';
```

Comment:
I got this

```diff
if (process.env.RSPACK_TITLE) {
+	process.title = process.env.RSPACK_TITLE;
+} else {
+	process.title = 'rspack';
+}
```

inside @rspack/cli/bin.js

Reply:
1. Is `process.env.RSPACK_TITLE` necessary? Users can override `process.title` in `rspack.config.ts` without the need for additional env variable.
2. I add `-node` suffix to show that this is a Node.js process.